src/pages/Protocol.vue

```diff
@@ -1,115 +1,32 @@
 <template>
 	<www-page>
 		<div id="banner-container">
-			<div id="banner-content">
-				<div id="banner-text">
+			<div id="banner-content" class="tw-bg-primary">
+				<p id="banner-text" class="tw-text-h1">
 					Building the credit bureau of the future
-				</div>
+				</p>
 				<div id="banner-logo">
 					<img id="banner-logo-image" src="~@/assets/images/protocol/kiva-protocol.svg">
 				</div>
 			</div>
 		</div>
 		<div class="row" id="intro">
-			<div class="small-12 columns">
-				<div id="intro-title">
+			<div class="small-12 columns tw-prose">
+				<p id="intro-title" class="tw-text-subhead">
 					{{ introTitle }}
-				</div>
+				</p>
 				<p v-for="(paragraph, index) in introText" :key="index">
 					{{ paragraph }}
 				</p>
 				<p><a href="https://pages.kiva.org/kiva-protocol-faq">{{ introLinkTitle }}</a></p>
 			</div>
 		</div>
-		<div v-if="showPartners" class="row" id="partners">
```

Comment:
This block has been turned off for 3 years

Reply:
Yea pretty sure we turned it off the first week lolz...